x-pack/test/security_solution_api_integration/test_suites/entity_analytics/risk_engine/trial_license_complete_tier/init_and_status_apis.ts

```diff
@@ -27,19 +27,44 @@ export default ({ getService }: FtrProviderContext) => {
   const es = getService('es');
   const supertest = getService('supertest');
   const kibanaServer = getService('kibanaServer');
+  const customSpaceName = 'ea-customspace-it';
   const riskEngineRoutes = riskEngineRouteHelpersFactory(supertest);
+  const riskEngineRoutesWithNamespace = riskEngineRouteHelpersFactory(supertest, customSpaceName);
   const log = getService('log');
 
-  // Failing: See https://github.com/elastic/kibana/issues/196319
-  describe.skip('@ess @serverless @serverlessQA init_and_status_apis', () => {
+  const createNamespace = async (spaceName: string) => {
+    const response = await supertest.post('/api/spaces/space').set('kbn-xsrf', 'true').send({
```

Comment:
I think we cant simplify this by using some of the testing services like the securityApi and spaces services. Check out this example: https://github.com/elastic/kibana/blob/e40011d1c145ef96beeb92bff097c9a60b6953a1/x-pack/test/security_solution_api_integration/test_suites/entity_analytics/risk_engine/trial_license_complete_tier/risk_scoring_task/task_execution_nondefault_spaces.ts